mappings/net/minecraft/class_5931.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5931
```

Comment:
Same, now this is `CaveSurfaceDecoratorConfig`

Reply:
Fixed in e762aacaed6c3345f0e94f4365d8d9d63f306ae8